packages/terra-core-docs/package.json

```diff
@@ -0,0 +1,95 @@
+{
+  "name": "@cerner/terra-core-docs",
+  "version": "1.10.0",
+  "description": "Contains documentation for packages in the terra-core monorepo",
+  "main": "index.js",
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git",
+    "directory": "packages/terra-core-docs"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-core-docs"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "engines": {
+    "node": ">=10.13.0"
+  },
+  "homepage": "https://github.com/cerner/terra-core",
+  "files": [
+    "lib",
+    "src"
+  ],
+  "scripts": {
+    "compile": "babel --root-mode upward src --out-dir lib --copy-files",
+    "lint": "eslint --ext .js,.jsx .",
+    "precompile": "rm -rf lib"
+  },
+  "dependencies": {
+    "terra-action-footer": "^2.60.0",
```

Comment:
i don't need all these dependencies for this package out of the gate, but we will need them all once we've migrated. These are just dependencies from terra-core but we can easily add any terra dependencies here.